aten/src/ATen/native/mps/operations/Convolution.mm

```diff
@@ -161,6 +161,16 @@ static Tensor _mps_convolution_impl(const Tensor& input_t,
   }
   TensorArg output{output_t, "result", 0};
 
+  // TODO: MPS convolution kernel currently does not support output channels > 2^16
+  for (auto elem : output_t.sizes()) {
+    TORCH_CHECK_NOT_IMPLEMENTED(
+        elem <= (1 << 16),
+        "Output channels > 65536 not supported at the MPS device. ",
+        "As a temporary fix, you can set the environment variable `PYTORCH_ENABLE_MPS_FALLBACK=1` ",
+        "to use the CPU as a fallback for this op. WARNING: this will be slower than running natively ",
+        "on MPS.");
```

Comment:
@jhavukainen bailing out here doesn't work because the op is already dispatched into the kernel, and we don't read the environment variable anymore at this point. Ref: #134416

Reply:
Yeah you are correct. Seem to have lost my train of thought there. Then the correct thing is to just raise the error that this is unimplemented as of now without the instruction to set the fallback env variable.